Getting-Started/Code/Source-Control/index.md

```diff
@@ -0,0 +1,139 @@
+However this can pose a problem in the common scenario where the line is blurred between an 'editor' and an 'implementor/developer' - where you might have individuals comfortable with making template changes via the backoffice, but not so comfortable with using a source control repository such as GIT.
+Umbraco Cloud is a great solution in these scenarios, as changes via the backoffice are tracked in a Git repository automatically - otherwise filewatchers and events, to alert when changes are made via the backoffice, or more simply, taking a copy of the production version of views into Source Control at the beginning of any development sprint.
+
+#### Macros
+
+Macros can be implemented in four different ways, and you do want to source control your changes to Macro implementations, so track the files in the following locations:
+
+- **Partial View Macros** - stored in /views/macropartials as .cshtml files
+- **Dynamic Razor** - stored in /macroScripts as .cshtml files
+- **UserControls** - stored in /userControls folder as .ascx and ascx.cs files
+- **XSLT** - stored in /xslt folder as .xslt files
+- 
+#### Controllers/Classes/Custom Code
+
+Any supporting custom code for your application should be in version control, eg any
+- C# implementation,
+  - Surface Controllers, 
+  - RenderMVCControllers,  
+  - ViewModels, 
+  - Helpers / Extension Methods, 
+  - Services etc
+- Supporting class library projects, 
+- App_Code, 
+- Models generated by Modelsbuilder in API or App_Data mode
+
+...all should be added to source/version control.
+
+#### Config
+
+Your site's /config folder contains the set of configuration files for your Umbraco site. Add these to source control along with config.transforms to change variables for different environments during deployment.
+
+#### DocumentType - Backoffice Structure Changes
+
+When you create and edit DocumentTypes, MediaTypes, DataTypes, Macros, DictionaryItems, Languages, MemberTypes & Templates('names' not the actual files) in the Umbraco Backoffice these values are stored in the Umbraco Database, making them very difficult to source control in a 'file based' version control system.
+
+However there is a third party package called *uSync* - https://our.umbraco.org/projects/developer-tools/usync/ which when installed will serialize these settings to disk, in a folder called /uSync - enabling you to version control these changes too, and import and apply to other Umbraco environments - read settings carefully for automatically importing changes - Also see [uSync Snapshots](https://our.umbraco.com/packages/developer-tools/usyncsnapshots/) for managing a release of a'set' of changes between environments.
```

Comment:
Can we have a point or 3 between all these words?  It makes it much more readable.  :-)